src/plugin/renderer/material_manager.cpp

```diff
@@ -30,6 +31,9 @@ wmr::MaterialManager::~MaterialManager()
 void wmr::MaterialManager::Initialize()
 {
 	auto& renderer = dynamic_cast<const ViewportRendererOverride*>(MHWRender::MRenderer::theRenderer()->findRenderOverride(settings::VIEWPORT_OVERRIDE_NAME))->GetRenderer();
+
+	LOG("Attempting to get a reference to the renderer.");
```

Comment:
It actually already tried to get a reference when this LOG is called 😉 Doesn't matter anyway

Reply:
That's true. Changed it!